mlprimitives/jsons/mlprimitives.candidates.dsp.Moments.json

```diff
@@ -0,0 +1,64 @@
+{
+    "name": "mlprimitives.candidates.dsp.Moments",
+    "author": "Rodrigo Diez <rjdiez@mit.edu>",
+    "documentation": "",
+    "description": "Anomalies detection in satellite telemetry data using a spectral mask",
+    "modalities": [],
+    "primitive": "mlprimitives.candidates.dsp.Moments",
+    "fit": {
+        "method": "fit",        
+        "args": [
+            {
+                "name": "X",
+                "description": "Training vector",
+                "type": "ndarray"
+            }
+        ]
+    },
+
+    "produce": {
+        "method": "produce",
+        "args": [
+            {
+                "name": "X",
+                "description": "Telemetry data vector",
+                "type": "ndarray"
+            }
+        ],
+        "output": [
+            {
+                
+                "name": "anomalies",
+                "descrtiption": "Data vector consisting of the anomalies detected in the telemetry data",
+                "type": "ndarray"
+            }
+        ]
+    },
+
+    "hyperparameters": {
+        "tunable": {
+            "gain": {
+                "description": "Multiplication factor used in the comparisson between the moments calculated using the training data and the moments of the telemetry data vector",
```

Comment:
"comparison" is misspelled.  Please fix other instances of "comparison" below as well.